### ISSUE TITLE
[FR] Add cross seed part of manage

### ISSUE THREAD

**Issue by ajhorvath20** [StuffAnThings/qbit_manage#55]
Hi, this is so good and would be great if cross seed is added into the image instead of running separately.

Thanks

**Comment by afraazali** [comment]
What would be the benefit to have cross seed running in the same image/container? That goes against the container principle too. Is there something not working for you by having cross-seed and qbit_manage running separately?

**Comment by bobokun** [comment]
I have to agree with @afraazali, it goes against how containers should be used.

**Comment by afraazali** [comment]
Just to add I have them working together in their own separate containers. If there’s anything I can help with, let me know.

**Comment by ajhorvath20** [comment]
Hi guys, thanks for looking into this.

No reason I just thought why two images so that's the reason I asked but thanks for your reply.
